packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/index.js

```diff
@@ -0,0 +1,87 @@
+import {
+  SubNav,
+  SubNavHeader,
+  SubNavLink,
+  SubNavLinkSection,
+  SubNavSection,
+  SubNavSections,
+  Box,
+} from '@strapi/parts';
+import { TextButton } from '@strapi/parts/TextButton';
+import { AddIcon } from '@strapi/icons';
+import upperFirst from 'lodash/upperFirst';
+import React from 'react';
+import { useIntl } from 'react-intl';
+import useContentTypeBuilderMenu from './useContentTypeBuilderMenu';
+import pluginId from '../../pluginId';
+
+// ! ASK SOUP ABOUT THE WAIT + TOGGLEPROMPT
+const wait = () => new Promise(resolve => setTimeout(resolve, 100));
```

Comment:
This was made for the alert in the Prompt to prevent from popping the alert when opening the modals. You can skip it for the moment.